roles/update-host/tasks/update-host.yml

```diff
@@ -1,12 +1,10 @@
 ---
 
 - name: "Update the host"
+  become: True
   package:
     name: "*"
     state: latest
   register: host_updated
-  when: 
-    - pkg_update|default(False) 
-  become: True
-
-
+  notify:
```

Comment:
This does't allow for a "forced reboot" (see global comment) that other users rely on from this role as-is